### DISCUSSION TITLE
Background color is dull as compared to gboard

### DISCUSSION THREAD

**Discussion by Ansarmughal99** [itsmartashub/floristyle#17] [General]
The background behind the keys are not up to mark as compared to the gboard keyboard The background looks white while the same color looks blue  in the gboard.
For refrenece see the screenshot
Pic 1:- gboard 
Pic 2 :- florisboard

![IMG_20241023_144132.jpg](https://github.com/user-attachments/assets/dae0243b-2117-4653-89d4-8b4d04a95f28)

Pic 2 :- Florisboard 
![IMG_20241023_144122.jpg](https://github.com/user-attachments/assets/cc6ff1bf-eaa3-46ba-bd4f-af7b7a3e349c)

**Comment by itsmartashub** [comment]
Thanks for providing these screenshots! I guess Gboard has updated its color shade in recent versions. Will check if this color for the background would look good for different Android versions and scenarios, and if so, will update it to darker.

**Reply by Unbaised99** [reply]
There is a need for an update of Colors in Material You theme. Most of the keyboard bg colors are not attractive like gboard or Futo.

**Reply by itsmartashub** [reply]
Please checkout [v5.0.0](https://github.com/itsmartashub/floristyle/releases/tag/v5.0.0) version
